bitcoin/src/pow.rs

```diff
@@ -269,6 +271,16 @@ impl From<CompactTarget> for Target {
     fn from(c: CompactTarget) -> Self { Target::from_compact(c) }
 }
 
+impl FromHexStr for CompactTarget {
+    type Error = ParseIntError;
```

Comment:
This is not carrying the input making error messages hard to understand. It should be OK to use `crate::parse::ParseIntError` instead.

Reply:
:+1: